addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -206,10 +206,18 @@
                 <t t-set="url" t-value="'/my/leads'"/>
                 <t t-set="placeholder_count" t-value="'lead_count'"/>
             </t>
-            <t t-call="portal.portal_docs_entry">
-                <t t-set="title">Opportunities</t>
-                <t t-set="url" t-value="'/my/opportunities'"/>
-                <t t-set="placeholder_count" t-value="'opp_count'"/>
+            <t t-if="not opp_count and request.env.user.partner_id.grade_id or request.env.user.commercial_partner_id.grade_id">
+                <a t-att-href="'/my/opportunities'" t-att-title="'Opportunities'" class="list-group-item list-group-item-action d-flex align-items-center justify-content-between">
+                    <t t-esc="'Opportunities'"/>
+                    <span class="badge text-bg-secondary rounded-pill" t-esc="'0'"/>
```

Comment:
I think you could write:

```suggestion
                    Opportunities
                    <span class="badge text-bg-secondary rounded-pill">0</span>
```
the second line is equivalent, but not having "Opportunities" in the t-esc allow it to be translatable (which it is not if it's in a t-esc)

but looking at how things are hidden, there seems to be a feature to keep line even with 0 count:

https://github.com/odoo/odoo/blob/0b27ac02e499bf15d9f011beda17ab202c1f7a42/addons/portal/static/src/js/portal.js#L102-L105

maybe it would make sense if it's what we want to also use that system, and have a similar code to:

https://github.com/odoo/odoo/blob/0b27ac02e499bf15d9f011beda17ab202c1f7a42/addons/sale/static/src/js/sale_portal.js#L10-L12

but that adds only `opp_count`.

Reply:
Thanks a lot @nle-odoo :ok_hand: